docs/changelog.md

```diff
@@ -15,6 +15,9 @@ permalink: /changelog/
 * **feature-addons**
   * Added optional `addonAllowPrivateBrowsingLabelDrawableRes` DrawableRes parameter to `AddonPermissionsAdapter.Style` constructor to allow the clients to add their own drawable. This is used to clearly label the WebExtensions that run in private browsing.
 
+* **browser-tabstray**
+  * Added optional `itemDecoration` DividerItemDecoration parameter to `BrowserTabsTray` constructor to allow the clients to add their own item decorations. This is used to ensure setting item decoration after setAdapter() is called.
```

Comment:
nit: We are only accepting `DividerItemDecoration`, instead of generic `listOf(ItemDecorations)`? The wording here implies the latter.

Reply:
I thought of using this constructor parameter just for adding an optional `DividerItemDecoration`, not implementations of `ItemDecoration`. I was referring to such a `DividerItemDecoration`, but I cn change the wording if it might be not clear enough. TY!